src/Native/Graphics/Element.js

```diff
@@ -39,7 +38,7 @@ Elm.Native.Graphics.Element.make = function(localRuntime) {
 			:
 				function(elementType)
 				{
-					var node = documentCreateElement(elementType);
+					var node = document.createElement(elementType);
```

Comment:
For some reason this was throwing an error in my browser (Chrome) - something along the lines of `documentCreateElement is not defined`. Changing it to this seemed to fix it.